### PR TITLE
activerecord: Remove unnecessary patches to other gems

### PR DESCRIPTION
**Pull request by tk0miya**

### CODE REVIEW

File: gems/activerecord/6.0/patch.rbs [hunk 1]
```diff
@@ -27,15 +27,3 @@ module PG
...
-# if the real types are available.
-class SimpleDelegator
-end
-
-# Remove the fake types for Gem::Version
-# if the real types are available.
-module Gem
-  class Version
-  end
-end
```
Comment (tk0miya) [inline gems/activerecord/6.0/patch.rbs:41]: https://github.com/ruby/rbs/blob/master/core/rubygems/version.rbs#L163

File: gems/activerecord/6.0/patch.rbs [hunk 2]
```diff
@@ -27,15 +27,3 @@ module PG
     def async_exec: (*untyped) -> untyped
   end
 end
-
-# Remove the fake SimpleDelegator type
-# if the real types are available.
-class SimpleDelegator
-end
```
Comment (tk0miya) [inline gems/activerecord/6.0/patch.rbs:34]: https://github.com/ruby/rbs/blob/master/stdlib/delegate/0/simple_delegator.rbs#L69

File: gems/activerecord/6.1/manifest.yaml
Comment (tk0miya) [inline gems/activerecord/6.1/manifest.yaml:1]: Is it better to copy the manifest.yaml? 7.* directories have their copy (not symlink).

Comment (ksss) [reply gems/activerecord/6.1/manifest.yaml:1]: Perhaps a copy is better.

**Review by ksss — COMMENTED** [review]
APPROVE

**Comment by tk0miya** [comment]
Now I replaced the symlink with a real file.

**Comment by ksss** [comment]
LGTM!

**Comment by tk0miya** [comment]
@ksss re-APPROVE is needed after the change. Could you review this again?
Thanks as always.

**Review by ksss — COMMENTED** [review]
APPROVE

**Pull request merged**